soccer/gameplay/play.py

```diff
@@ -32,3 +32,13 @@ def is_restart(cls):
     @classmethod
     def handles_goalie(cls):
         return False
+
+
+    ## Override to allow a play to run during the stopped state
+    # This is reserved only for very special plays that need to run during the
+    # stopped state.  Most plays should give up control during the stopped state
+    # to stopped.py, so stopped.py is the only play that should be overriding
+    # this.
+    @classmethod
+    def run_during_stopped(cls):
```

Comment:
Also at some point I should talk to you about functions versus just having properties. I would have just left his has a boolean field but it makes sense to have a function, so we could have it as like a state but still